sandbox/tilted_bbl.jl

```diff
@@ -0,0 +1,128 @@
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.TurbulenceClosures: AnisotropicMinimumDissipation
+using CUDA
+
+const Nx = 256; const Lx = 1000
+const Nz = 32; const Lz = 100
+const θ_rad = 0.05 # radians
+const θ_deg = rad2deg(θ_rad) # degrees
+const N²∞ = 1e-5
+const V∞ = 0.1
+const g̃ = (sin(θ_rad), 0, cos(θ_rad))
+
+#++++ Grid
+S = 1.3
+z_faces(k) = Lz*(1 + tanh(S * ( (k - 1) / Nz - 1)) / tanh(S))
+topo = (Periodic, Periodic, Bounded)
+grid = VerticallyStretchedRectilinearGrid(topology=topo,
+                                          architecture = CUDA.has_cuda() ? GPU() : CPU(), 
+                                          size=(Nx, 1, Nz),
+                                          x=(0, Lx), y=(0, 6*Lx/Nx), z_faces=z_faces,
+                                          halo=(3,3,3),
+                                         )
+println(); println(grid); println()
+#----
+
+#++++ Buoyancy model and background
+buoyancy = Buoyancy(model=BuoyancyTracer(), vertical_unit_vector=g̃)
+tracers = :b
+
+b∞(x, y, z, t) = N²∞ * (x*g̃[1] + z*g̃[3])
+B_field = BackgroundField(b∞)
+
+b_bottom(x, y, t) = -b∞(x, y, 0, t)
+grad_bc_b = GradientBoundaryCondition(b_bottom)
+b_bcs = TracerBoundaryConditions(grid, bottom = grad_bc_b)
+#----
+
+#+++++ Boundary Conditions
+#+++++ Bottom Drag
+const z₀ = 0.01 # roughness length (m)
+const κ = 0.4 # von Karman constant
+const cᴰ = (κ / log(grid.zᵃᵃᶜ[1]))^2 # quadratic drag coefficient
```

Comment:
```suggestion
z₁ = znodes(Center, grid)[1]
const cᴰ = (κ / log(z₁ ))^2 # quadratic drag coefficient
```

to make this robust against changes to grid properties (and also so it works for other grids). You may need to add `using Oceananigans.Grids: znodes`. It may also be good to convert to `Array` so that this example runs on the GPU.